packages/ui/src/components/modals/ConditionModal.tsx

```diff
@@ -82,10 +82,10 @@ interface ComparisonTypeOption extends OF.IDropdownOption {
     data: CLM.ComparisonType
 }
 
-const constructComparisonTypeDropdownOptions = (options: CLM.ComparisonType[]): ComparisonTypeOption[] => {
+const convertComparisonTypeToDropdownOption = (options: CLM.ComparisonType[]): ComparisonTypeOption[] => {
     return options.map((comparisonType: string) => {
         let conditionText = `unknown`
-        if (comparisonTypeDisplay ?.[comparisonType]) {
+        if (comparisonTypeDisplay?.[comparisonType]) {
```

Comment:
Don't think you need the optional chaining here. `comparisonTypeDisplay` is a constant that will always be defined. Means remove the `?`

Reply:
What was resolution on this?

Reply:
removed